Tiltfile

```diff
@@ -34,6 +34,25 @@ k8s_resource("opensearch-dashboards", port_forwards="7500:5601")
 
 k8s_resource("temporal-ui", port_forwards="7440:8080")
 
+local_resource(
+  "gen-dashboard-client",
+  cmd="docker container run \
+    --rm \
+    --user $(id -u):$(id -g) \
+    --volume $(pwd):/local \
+    openapitools/openapi-generator-cli:v6.0.0 generate \
+      --input-spec /local/internal/api/gen/http/openapi.json \
+      --generator-name typescript-fetch \
+      --output /local/dashboard/src/openapi-generator/ \
+      --skip-validate-spec \
+      -p 'generateAliasAsModel=false' \
+      -p 'withInterfaces=true' \
+      -p 'supportsES6=true' \
+  ",
+  deps=["internal/api/gen/http/openapi.json"],
```

Comment:
Do we need to watch for changes in other files or just in `http/openapi.json` file?